contribs/docker/flask/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:slim-buster
+
+RUN apt-get update -y && apt-get dist-upgrade -y
```

Comment:
Why doing this command?
Base image should be up-to-date and we should build this image with `docker build --pull` flag

Reply:
You're right, this `RUN` line is not useful. As I'm using python:slim-buster (not locked to a specific version of Python), the base image is up-to-date already and is regularly updated.